### ISSUE TITLE
Publish to PyPI

### ISSUE THREAD

**Issue by ReneVolution** [ReneVolution/profanity-omemo-plugin#7]
To make adoption even easier, the package should be available on PyPI. Let's see how the post install etc. can work there.

**Comment by mvhulten** [comment]
Yes, this would be a great enhancement.  In PyPI it should have a complete dependency tree as well, which would make installation much easier.
